taxonomies/food/categories.txt

```diff
@@ -78940,7 +78930,7 @@ ja: ブラバスソース
 nl: Brava sauzen
 wikidata:en: Q2102385
 
-< en:Tomato sauces
+< en:Sauces
```

Comment:
Can you include a motivation in your PR for moving ketchup from tomato sauces to generic sauces?

Reply:
Because < en:Tomato sauces should apply to "Tomato ketchup" and not "ketchup". I just added "tomato sauces" to "tomato ketchup".